pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -112,7 +115,9 @@ type PipelineRef struct {
 type PipelineRunStatus struct {
 	duckv1beta1.Status `json:",inline"`
 
-	// In #107 should be updated to hold the location logs have been uploaded to
+	// Deprecation Notice: The field Results will be deprecated in v0.8.0
```

Comment:
`will be deprecated in v0.8.0` -> `will be removed in v0.8.0`.  I think we're technically deprecating it in 0.7.0 by making the announcement :)

Same comment for the other lines/files where this notice appears.

Reply:
Thanks for catching that. Changed to `will be removed in v0.8.0.`